doc/cookbook/error_handler.rst

```diff
@@ -36,3 +36,18 @@ the ``$debug`` argument::
     use Symfony\Component\Debug\ExceptionHandler;
 
     ExceptionHandler::register(false);
+
+
+.. note::
+
+    Important caveat when using Silex in a command-line interface:
+    The ``ExceptionHandler`` should not be enabled as it would convert an error
+    to HTML output and return a non-zero exit code.
+
+    .. code-block:: php
+
+        use Symfony\Component\Debug\ExceptionHandler;
+
+        if (!in_array(PHP_SAPI, ['cli', 'phpdbg'])) {
```

Comment:
What about `cli-server` here?

Reply:
@xabbuh If you mean the built-in PHP server meant for development, it shouldn't be included here as it's alright converting error to HTML in that SAPI.